test_requests_futures.py

```diff
@@ -4,10 +4,12 @@
 """Tests for Requests."""
 
 from concurrent.futures import Future
-from requests import Response, session
 from os import environ
+import unittest
+import sys
```

Comment:
Stylistically I prefer everything to be `from X import Y, Z` rather than `import X`... `X.whatever`